spec/food_order_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "food_order"
+require "menu"
+require "text_client"
+
+describe FoodOrder do
+
+  let(:dish_list) { { "test dish1" => 6.00, "test dish2" => 4.00, "test dish3" => 4.00 } } 
+  let(:menu_double) { instance_double(Menu, menu: dish_list) }
+
+  let(:formatted_time) { (Time.now + 3600).strftime("%k:%M") }
+  let(:formatted_confirmation) { "Thank you! Your order was placed and will be delivered before #{formatted_time}" }
+
+  let(:text_client_double) { instance_double(TextClient) }
+
+  subject { described_class.new(menu_double, text_client_double) }
+
+  context "placing an order" do
```

Comment:
clear and simple code complies with DRY principles

Reply:
Actually, DRY is less important for tests - readability of individual tests is the most important.